packages/camels/camels.0.0.3/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: ["maintenance@identity-function.com"]
+authors: ["yomimono"]
+homepage: "https://github.com/yomimono/camels"
+bug-reports: "https://github.com/yomimono/camels/issues"
+dev-repo: "git+https://github.com/yomimono/camels.git"
+license: "GPLv3"
+tags: ["game"]
+
+build: [
+  ["dune" "build"]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```
`dune build` alone without `-p` or `--profile=release` is going to break at some point because it is meant for development and turns some warnings as errors.